src/Text/Toml/Parser.hs

```diff
@@ -5,5 +5,21 @@ module Text.Toml.Parser
     ( parseToml
     ) where
 
-parseToml = undefined
+import Text.Toml.Tokenizer
+import Text.Toml.Types.Tokens
+import Text.Toml.Types.Toml
 
+import Control.Monad ((<=<))
+import Data.Text (Text)
+import Data.Monoid ((<>))
+
+parseToml :: Text -> Either String Toml
+parseToml = interpret empty <=< tokenize
+
+interpret :: Toml -> [Token] -> Either String Toml
+interpret toml [] = Right toml
+interpret toml (IdentifierT _ n:EqualT _:KeywordTrueT _:rest) =
+    interpret (insert n (TBoolean True) toml) rest
+interpret toml (IdentifierT _ n:EqualT _:KeywordFalseT _:rest) =
+    interpret (insert n (TBoolean False) toml) rest
+interpret _ t = Left $ "Unexpected token: " <> show t
```

Comment:
Very glad for the start. Only suggestion is that at the referred lines, why not use/extend the combinators in `src/Text/Toml/Combinators.hs`? The purpose of using them would be to improve error reporting and make the code easier to follow. 

So instead of pattern-matching, you can do e.g., `identifier <*> (true <|> false)`